docs/admin/production_deployment.md

```diff
@@ -33,3 +33,22 @@ You call then call the `api-gateway` service at the following path: `/actuator/i
 ```
 
 Be aware that, once activated, this endpoint will be publicly accessible. If this is not what you want (and it should not be, for a production deployment), be sure to add access restriction to this endpoint (for instance, only allowing a monitoring infrastructure).
+
+
+## Launching Stellio with a different port binding
```

Comment:
I am a bit worried because it is mixing two things: 1. launch Stellio using different ports bindings (using a "context source" environment) 2. launch a second instance of Stellio acting as a context source

So I would split this into two things:
- how to launch Stellio using different ports (without using anything from the context source things), in `misc_configuration_tips.md`
- how to launch a second instance of Stellio to act as a context source (maybe in a new page since it is expected to grow over time?)